dbquery/pagination/tests/Test.js

```diff
@@ -52,30 +52,69 @@ describe(testDescription, function () {
       operationName: "Customers",
       variables: {
         first: 2,
-        after: CURSOR
+        after: CURSOR,
       },
       expected: {
         customers: {
           edges: [
             {
               node: {
                 id: "3",
-                name: "Salim Ali                                         "
-              }
+                name: "Salim Ali                                         ",
+              },
             },
             {
               node: {
                 id: "4",
-                name: "Jane Xiu                                          "
-              }
-            }
+                name: "Jane Xiu                                          ",
+              },
+            },
           ],
           pageInfo: {
             endCursor: "eyJjIjoiTDpRdWVyeTpjdXN0b21lcnMiLCJvIjozfQ==",
-            hasNextPage: true
+            hasNextPage: true,
+          },
+          totalEdges: 10,
+        },
+      },
+    },
+    {
+      label: "lookup-filter",
+      query: requests,
+      operationName: "Lookup",
+      variables: {
+        first: 2,
+        f: {
+          or: {
+            name: { eq: "John Doe" },
+            email: { eq: "jane.smith@example.com" },
+          },
+        },
+      },
+      expected: {
+        customers: {
```

Comment:
this should be changed to `lookup` (hence the test failure)  as per this operation https://github.com/stepzen-dev/snippets/pull/92/changes#diff-17e39791f40afabc41c58a04a19272d9ff2f95be98c67c8de5a3f37e03f3482cR20